lua/autorun/sh_portal_movement.lua

```diff
@@ -168,6 +183,8 @@ hook.Add("Move", "seamless_portal_teleport", function(ply, mv)
 			local exitPortal = hitPortal:GetExitPortal()
 			local editedPos, editedAng = SeamlessPortals.TransformPortal(hitPortal, exitPortal, tr.HitPos, ply:EyeAngles())
 			local _, editedVelocity = SeamlessPortals.TransformPortal(hitPortal, exitPortal, nil, mv:GetVelocity():Angle())
+			-- What the fuck is this?
+			-- TODO: Comment what this does
```

Comment:
if a prop is under a certain velocity, it will normalize it so it doesnt start rapidly teleporting if 2 portals are facing up